frankenphp/build-solidinvoice.sh

```
@@ -0,0 +1,267 @@
+# ============================================================================
+# PHP Configuration
+# ============================================================================
+
+if [ -z "${PHP_VERSION:-}" ]; then
+	export PHP_VERSION="8.5"
+fi
+
+# SolidInvoice-specific extensions (removed problematic/unnecessary ones)
+
+if [ -z "${PHP_EXTENSIONS:-}" ]; then
+    export PHP_EXTENSIONS=$(cat ./build-static.sh | grep 'defaultExtensions="' | head -n 1 | sed 's/.*defaultExtensions="//' | sed 's/".*//' | tr ',' '\n' | grep -v -E 'ssh2|gmp|pdo_sqlsrv|memcache|memcached' | tr '\n' ',' | sed 's/,$//')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)